binding_generator.py

```diff
@@ -1702,7 +1702,7 @@ def correct_type(type_name, meta=None):
     type_conversion = {"float": "double", "int": "int64_t", "Nil": "Variant"}
     if meta != None:
         if "int" in meta:
```

Comment:
That won't match `bool`, is it handled differrently?

Reply:
Moved it to the different place, now it should handle `bool`, and won't unnecessarily change function signatures.